src/widget/woverview.cpp

```diff
@@ -85,7 +85,7 @@ WOverview::WOverview(
     m_pPassthroughControl =
             new ControlProxy(m_group, "passthrough", this, ControlFlag::NoAssertIfMissing);
     m_pPassthroughControl->connectValueChanged(this, &WOverview::onPassthroughChange);
-    onPassthroughChange(m_pPassthroughControl->get());
+    m_bPassthroughEnabled = static_cast<bool>(m_pPassthroughControl->get());
```

Comment:
I agree that in this special case introducing a private, non-virtual function does not make any sense.